src/linalg.jl

```diff
@@ -1,15 +1,3 @@
-# declare struct 
```

Comment:
@jieli-matrix when you write a comment, do not repeat what the codes say, try to explain the intentions of the code.

"declare struct" is a useless comment. Why we need this if-block is a useful comment.